### PR TITLE
Detailed prototype chosen activities page

### PR DESCRIPTION
**Pull request by kiwikid747**

### CODE REVIEW

File: app/locales/en.json [hunk 1]
```diff
@@ -40,6 +40,12 @@
     "overview": "How interested are you in this activity?",
     "backButton": "Back"
   },
+  "chosen-activities": {
+    "pageTitle": "Choose your activities",
+    "backButton": "Back",
+    "heading": "Your chosen activities",
+    "subheading": "Get better at my applications and interviews"
```
Comment (mangogrin) [inline app/locales/en.json]: shouldn't this be a group title?

File: app/locales/en.json [hunk 2]
```diff
@@ -234,4 +240,4 @@
     "action-2": "try them out",
     "button-label": "Go to my activities"
   }
-}
+}
```
Comment (mangogrin) [inline app/locales/en.json]: add a new line here

File: app/views/chosen-activities.mustache [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  <div class="grid-row">
+    <div class="column-two-thirds">
+
+      <h1 class="heading-xlarge">
+        <span class="category heading-secondary">{{#__}}chosen-activities.subheading{{/__}}</span>
+        {{#__}}chosen-activities.heading{{/__}}
+      </h1>
+
+      <div class="part-navigation-container">
+        <nav role="navigation" class="app-c-contents-list" aria-label="Categories for this group">
```
Comment (mangogrin) [inline app/views/chosen-activities.mustache]: Texts should go to locale file

File: app/views/chosen-activities.mustache [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+          <h2 class="font-xsmall">Contents:</h2>
+          <div class="grid-row">
+            <div class="column-two-thirds">
+              <ol class="list list-contents app-c-contents-list__list" id="contents">
+                {{#categories}}
+                  {{#hasActivities}}
+                    {{#isSelectedCategory}}
+                      <li class="font-xsmall app-c-contents-list__list-item app-c-contents-list__list-item--dashed app-c-contents-list__list-item--active" aria-current="true">
+                       {{title}} ({{numActivities}} {{#moreThanOneActivity}}activities{{/moreThanOneActivity}}{{^moreThanOneActivity}}activity{{/moreThanOneActivity}})
```
Comment (mangogrin) [inline app/views/chosen-activities.mustache]: i think this logic should be problem should be solved by the time view model is created

File: app/views/chosen-activities.mustache [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+        </nav>
+      </div>
+
+    {{#currentCategory}}
+      <h2 class="heading-large" data-test="category-heading">{{title}}</h2>
+
+      {{#activities}}
+        <div class="{{version}} chosen-activities-heading {{name}}">
+          <h3 class="heading-medium">{{ title }}</h3>
+          {{!--<p class="font-xsmall"><a href="">Change</a></p>--}}
```
Comment (mangogrin) [inline app/views/chosen-activities.mustache]: redundant comment?

File: test/integration/chosen-activities.spec.js [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+      );
+
+      it(`should contain valid google tag manager data for version ${version}`, () =>
+        pageUnderTest.visit(version, accountId, 'GRP-3').then(() =>
+          expect(googleTagManagerHelper.getUserVariable()).to.equal(accountId)
+        )
+      );
+    });
+
+    // it(`should have back button for version ${version}`, () =>
```
Comment (mangogrin) [inline test/integration/chosen-activities.spec.js]: uncomment or remove

File: test/integration/chosen-activities.spec.js [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+
+const accountId = uuid.v4();
+
+describe('chosen activities page', () => {
+  describe('page outline', () => {
+    ['a', 'b'].forEach((version) => {
+      before(() =>
+        helper.cleanDb()
+          .then(() => helper.saveAllActivitiesAsSorted(accountId))
+          .then(() => pageUnderTest.visit('c', accountId))
```
Comment (mangogrin) [inline test/integration/chosen-activities.spec.js]: looks like this is not needed. If needed change it to `pageUnderTest.visit(version,...`

File: app/locales/en.json (hunk 1 above)
Comment (will518) [reply app/locales/en.json]: Yes. Great spot. Thanks.

**Comment by mangogrin** [comment]
If I click on the other links like ` I'd like help trying this (1 activity)` it shows almost an empty page

File: app/views/chosen-activities.mustache (hunk 5 above)
Comment (will518) [reply app/views/chosen-activities.mustache]: I left this in for the next piece of work to add a change link.

**Comment by will518** [comment]
👍

**Pull request merged**